content/source/assets/javascripts/analytics.js

```diff
@@ -13,4 +13,25 @@ document.addEventListener('turbolinks:load', function() {
       product: 'terraform'
     }
   })
+
+  track('.notification', function(el) {
+    var params = {
+      name: 'Alert Banner',
+      variant: getVariantTextFromHtml(el.innerHTML)
+    }
+    // Create a stringified `label` prop of the whole event for GA tracking purposes
+    var label = JSON.stringify(params)
+    params.label = label
+    params.event = 'CTA Clicked'
+    // Return the whole event - it will be parsed by the global `track()` function - See Segment debugger to verify
+    return params
+  })
 })
+
+// Variants for some events may contain a mix of HTML and the Text we're testing, this strips the HTML
+function getVariantTextFromHtml(html) {
+  var el = document.createElement('div')
+  el.innerHTML = html
+  var text = el.textContent || el.innerText || ''
+  return text.replace(/\s+/g, ' ').trim()
+}
```

Comment:
I probably would have not abstracted `getVariantTextFromHtml` into another function, as it's not doing anything terribly complicated & it's not something we want used in any other place.

Either that, or I would have given it a much more specific name (`getVariantTextFromAlertBannerHtml`), to ensure that it's very clear that this is not a generic function that can be used elsewhere.

Reply:
Yep - well turns out I overcomplicated it! Will be nixing the function